docs/discovery-search.html

```diff
@@ -493,9 +494,11 @@
 <!-- results column -->
 <div class="nypl-column-three-quarters" id="mainContent">
       <div class="nypl-results-summary" aria-live="assertive" aria-atomic="true" role="presentation">
-        <span class="nypl-results-count">9,478</span> results found for
-        <span class="nypl-facet">keywords <strong>new york waterways<button><svg class="nypl-icon" preserveAspectRatio="xMidYMid meet" viewBox="0 0 10 10" aria-hidden="true"><title>times.icon</title><polygon points="2.3,6.8 3.2,7.7 5,5.9 6.8,7.7 7.7,6.8 5.9,5 7.7,3.2 6.8,2.3 5,4.1 3.2,2.3 2.3,3.2 4.1,5 "/></svg><span>Remove new york waterways keyword filter</span></button></strong></span>,
-        <span class="nypl-facet">subject <strong>Poems<button><svg class="nypl-icon" preserveAspectRatio="xMidYMid meet" viewBox="0 0 10 10" aria-hidden="true"><title>times.icon</title><polygon points="2.3,6.8 3.2,7.7 5,5.9 6.8,7.7 7.7,6.8 5.9,5 7.7,3.2 6.8,2.3 5,4.1 3.2,2.3 2.3,3.2 4.1,5 "/></svg><span>Remove Poems subject filter</span></button></strong></span>.
+        <h2>
```

Comment:
well it is a matter of opinion... @wlla and i agree it is a heading... stanford [agrees it is a heading](https://searchworks.stanford.edu/?f[access_facet][]=At+the+Library)… others may not see it that way... i think it is informative to know how many results are and knowing exactly what you are faceting on... lets you decide if you facet further or not... i really want to get v15 out before the demo but it is up to you… if we have those results up there, they should be a heading... otherwise we must decrease the prominence altogether